docs/license-handling.asciidoc

```diff
@@ -0,0 +1,63 @@
+[id="{p}-licensing"]
+== Managing licenses in ECK
+
+When you install the default distribution of ECK, you receive a basic license. Any Elastic stack application you manage through
+ECK will also be basic licensed. For the full list of free features that are included in the basic license, see: https://www.elastic.co/subscriptions
+
+[float]
+=== Starting a trial
+If you want to try the features included in the Enterprise subscription, you can start a 30-day trial. To start a trial create
+a Kubernetes secret as shown below:
+
+[source,yaml]
+----
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: any-name-you-like
+  namespace: elastic-system
```

Comment:
I'm wondering if it's necessary to add a note explaining that this must be the namespace in which the/a global operator is deployed.

Reply:
It does not have to be currently. Any namespace will work. We can discuss whether we need to change that though in another issue.